tcl/handlers.tcl

```diff
@@ -3,14 +3,14 @@ namespace eval qc::handlers {
     namespace export call exists validate2model validation
     namespace ensemble create
 
-    proc call {path method} {
+    proc call {method path} {
```

Comment:
### Typical ensemble style
#### Pros
* Organisation/tidyness - This is the only reason that I can see as to why this style is usually used for ensembles.

#### Cons
* Currently, tags don't account for ensemble procs so they cannot be searched using `tags` commands in emacs.